internal/build/manager.go

```diff
@@ -22,5 +22,6 @@ type Result struct {
 //go:generate mockgen -source=manager.go -package=build -destination=mock_manager.go
 
 type Manager interface {
+	IsOutputPresent(ctx context.Context, mod kmmv1beta1.Module, containerImage string) (bool, error)
```

Comment:
Why do we need this separate API in the Build Interface. Why not use directly the registry interface? Logically checking image existence is not related to build, we can use this API  in other flows also (not related to build)